packages/private/edtr-io/src/plugins/solution-steps/index.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  child,
+  object,
+  list,
+  string,
+  boolean,
+  EditorPluginProps,
+  EditorPlugin
+} from '@edtr-io/plugin'
+
+import { SolutionStepsEditor } from './editor'
+import * as Guidelines from './helper/guideline-texts'
+
+function createSolutionStepState(
```

Comment:
add custom serializer / custom state type so that `hasStrategy` etc. is not part of state